knative-operator/pkg/controller/add_knativekafka.go

```diff
@@ -0,0 +1,11 @@
+package controller
+
+//import (
+//	"github.com/openshift-knative/serverless-operator/knative-operator/pkg/controller/knativekafka"
+//)
+
+func init() {
+	// AddToManagerFuncs is a list of functions to create controllers and add them to a manager.
+	// TODO: temp change to disable KnativeKafka reconciliation
+	// AddToManagerFuncs = append(AddToManagerFuncs, knativekafka.Add)
+}
```

Comment:
Disabled for now since the CRD is not installed with the CSV yet. I will revert later once the CSV is adjusted and the controller actually does something with the CRs.